client/components/DataManagement/CoveredAtListDialogWithButton/index.jsx

```diff
@@ -0,0 +1,68 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { Button } from 'react-bootstrap';
+import BasicModal from '../../common/BasicModal';
+import { NoneText } from '../DataManagementRow';
+
+const CoveredAtListDialogWithButton = ({ ats }) => {
+    const [show, setShow] = useState(false);
+    const handleShow = () => setShow(true);
+    const handleClose = () => setShow(false);
+
+    const renderButtonLabel = () => {
+        return (
+            <span>
+                <strong>{ats.length} Desktop</strong> Screenreaders
```

Comment:
Would it be possible to make this two words?

I know this probably clashes with the specified design. Just to back up the suggestion a little bit, I googled `"screen readers"` and got 80 million results, and then googled `"screenreaders"` and got 200K results. In other words, the term screen readers as two words is 400 times as common.